lib/OpenQA/Schema/Result/JobGroupParents.pm

```diff
@@ -0,0 +1,72 @@
+# Copyright (C) 2016 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+package OpenQA::Schema::Result::JobGroupParents;
+use base qw/DBIx::Class::Core/;
+use strict;
+
+__PACKAGE__->table('job_group_parents');
+__PACKAGE__->load_components(qw/Timestamps/);
+
+__PACKAGE__->add_columns(
+    id => {
+        data_type         => 'integer',
+        is_auto_increment => 1,
+    },
+    name => {
+        data_type   => 'text',
+        is_nullable => 0,
+    },
+    default_size_limit_gb => {
+        data_type     => 'integer',
+        default_value => 100,
+    },
+    default_keep_logs_in_days => {
+        data_type     => 'integer',
+        default_value => 30,
+    },
+    default_keep_important_logs_in_days => {
+        data_type     => 'integer',
+        default_value => 120,
```

Comment:
I still think we should not have the default values in the database, but rather have a possibility to change the default values later by changing the code. If you change the default value then the migration will hardcode this in the database and you'll never know if the admin meant to store 120 or leave the default.